boards/shields/nrf700x_nrf54h20dk/nrf700x_nrf54h20dk.overlay

```diff
@@ -76,6 +77,23 @@
 			reg = <0x126000 DT_SIZE_K(64)>;
 		};
 	};
+	cpuapp_rw_partitions: cpuapp-rw-partitions {
+		compatible = "nordic,owned-partitions", "fixed-partitions";
+		status = "okay";
+		perm-read;
+		perm-write;
+		perm-secure;
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		dfu_partition: partition@136000 {
+			reg = < 0x136000 DT_SIZE_K(676) >;
```

Comment:
Just FYI: since `dfu_partition` is smaller than `cpuapp_slot0_partition` and the compression is not yet supported, it is not guaranteed that the application will be updateable. The SDFW bundle will fit.